medusa/search/proper.py

```diff
@@ -247,6 +247,14 @@ def _get_proper_results(self):  # pylint: disable=too-many-locals, too-many-bran
                     if best_result.name not in processed_propers_names:
                         self.processed_propers.append({'name': best_result.name, 'date': best_result.date})
                     continue
+                streaming_service = NameParser()._parse_string(release_name).guess.get(u'streaming_service')
```

Comment:
Please don't parse again, use the parsed data from https://github.com/pymedusa/Medusa/blob/c710dca14dcaefc3d953b1379a7b2cc233c1cb90/medusa/search/proper.py#L242

Reply:
I was just commiting that. thanks